dts/bindings/dma/st,stm32-dmamux.yaml

```diff
@@ -55,10 +54,11 @@ description: |
     for client SPI of stm32wb55x
      spi1: spi@40013000 {
          compatible = "st,stm32-spi";
-         dmas = <&dmamux1 11 7 0x20440 0
-         &dmamux1 1 6 0x20480 0>;
+         dmas = <&dmamux1 11 7 0x20440
+         &dmamux1 1 6 0x20480>;
```

Comment:
Xould be nice to fix indentation, while we're at it

Reply:
done